.rubocop.yml

```diff
@@ -95,3 +95,7 @@ Lint/SuppressedException:
 # Enforce LF line endings, even when on Windows
 Layout/EndOfLine:
   EnforcedStyle: lf
+
+Layout/HashAlignment:
+  EnforcedHashRocketStyle: [key, table]
```

Comment:
I added the table style as an acceptable layout so the options hashes were easier to read.